### PR TITLE
[MRG] Test TUH classes

### PR DESCRIPTION
**Pull request by gemeinl**

### CODE REVIEW

**Comment by gemeinl** [comment]
I am thinking to also add this kind of mocking to the `tuh_eeg_corpus.py` example, such that it can be automatically run by CI / docs. This would ensure that there are no bugs in the example.
Are there any objections regarding this or support this idea @agramfort @robintibor @hubertjb @sliwy ?

**Comment by robintibor** [comment]
wow looks superprofessional to me

**Review by robintibor — APPROVED** [review]

**Comment by robintibor** [comment]
really impressive

**Review by agramfort — APPROVED** [review]
nice !

**Comment by robintibor** [comment]
Please mark [WIP] or [MRG] depending on what you want @gemeinl

**Comment by gemeinl** [comment]
Done from my side although the docs seem to have died somewhere during this PR...

File: examples/plot_tuh_eeg_corpus.py [hunk 1]
```diff
@@ -64,14 +116,15 @@
     alpha=.5,
 )
 ax.legend(genders)
+ax.set_xlabel('Age [years]')
+ax.set_xlabel('Count')
```
Comment (robintibor) [inline examples/plot_tuh_eeg_corpus.py:120]:
```suggestion
ax.set_ylabel('Count')
```

Mistake in axis label...

**Review by robintibor — CHANGES_REQUESTED** [review]

**Comment by gemeinl** [comment]
List in https://github.com/braindecode/braindecode/pull/276/files#diff-60801eb2489f88511279735603d520c5d613d4daa1f6a9630d0b681830cf72beR197-R203 still does not look as intended. Apart from this finally done.

File: examples/plot_tuh_eeg_corpus.py [hunk 2]
```diff
@@ -138,13 +194,13 @@ def select_by_channels(ds, ch_mapping):
...
 # `mne`. Data will be loaded by the first preprocessor that has a mention of it
 # in brackets:
-# - crop the recordings to a region of interest
-# - re-reference all recordings to 'ar' (requires load)
-# - rename channels to short channel names
-# - pick channels of interest
-# - scale signals to microvolts (requires load)
-# - resample recordings to a common frequency (requires load)
-# - create compute windows
+# #. crop the recordings to a region of interest
```
Comment (robintibor) [inline examples/plot_tuh_eeg_corpus.py:197]:
```suggestion
#
# #. crop the recordings to a region of interest
```

**Review by robintibor — APPROVED** [review]

**Comment by robintibor** [comment]
So you needed to add one line only with:
 `#`
right before the list for everything to work @gemeinl 
Thanks for the great work, merged!

**Comment by gemeinl** [comment]
Ah, I see. Same mistakes over and over again...
Thanks for fixing!

**Pull request merged**